ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -181,15 +199,46 @@ private[apiserver] final class ApiSubmissionService private[services] (
 
   private def handleCommandExecutionResult(
       result: Either[ErrorCause, CommandExecutionResult]
-  ): Future[CommandExecutionResult] =
+  ): Future[CommandExecutionResult] = {
+
+    def toStatusException_V1(errorCause: ErrorCause): StatusRuntimeException =
```

Comment:
```suggestion
    def toStatusExceptionV1(errorCause: ErrorCause): StatusRuntimeException =
```
Can you also use camelCase all the way?

Reply:
Ok